src/test/java/com/udojava/evalex/TestContinuousUnary.java

```diff
@@ -0,0 +1,35 @@
+package com.udojava.evalex;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+/**
+ * Created by samo on 2019/3/29.
+ *
+ * @author samo
+ * @date 2019/03/29
+ */
+public class TestContinuousUnary {
```

Comment:
Could you add additional tests for stacking of mutiple operators? For example, what is the expected output of `-+---+++--++-1` and `1--++++---2+-+----1`?

Reply:
I take your advice, then push another commit.